Makefile

```diff
@@ -114,20 +119,20 @@ unittest:
 
 .PHONY: common-deps-update
 common-deps-update:	controller-gen kustomize
-	go mod tidy
+	go mod tidy && go mod vendor
 
 .PHONY: shellcheck
-shellcheck: ## Run shellcheck
+shellcheck: ## Run shellcheck.
 	@echo "Running shellcheck"
 	hack/shellcheck.sh
 
 .PHONY: bashate
-bashate: ## Run bashate
+bashate: ## Run bashate.
 	@echo "Running bashate"
 	hack/bashate.sh
 
 .PHONY: ci-job	
-ci-job: common-deps-update generate fmt vet lint golangci-lint unittest shellcheck bashate update-bindata bundle-check #test
+ci-job: common-deps-update generate fmt vet lint golangci-lint unittest shellcheck bashate bundle-check imager-unittest
```

Comment:
why is update-bindata removed?

Reply:
The update-bindata is required here as part of checking that generated code has been updated - at least until (and if) we get rid of all the bindata bits and implement them fully in go, so it would need to be restored (and done ahead of "bundle-check")